_includes/comparison.html

```diff
@@ -230,7 +230,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://istio.io/docs/setup/install/multicluster/" target="_blank">yes</a></td>
         <td><a href="https://linkerd.io/2/features/multicluster/" target="_blank">yes</a></td>
         <td></td>
-        <td><a href="https://www.consul.io/docs/connect/gateways/mesh-gateway" target="_blank">yes</a></td>
+        <td><a href="https://www.consul.io/docs/k8s/installation/multi-cluster/kubernetes" target="_blank">yes</a></td>
```

Comment:
Would https://www.consul.io/docs/k8s/installation/multi-cluster be the better link?

Reply:
That works. I updated the URL to point to the multi-cluster overview page.